cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/namespace.rst

```diff
@@ -124,10 +124,10 @@ To edit an existing namespace, submit an HTTP PUT request to::
 
    * - Parameter
      - Description
-   * - ``<namespace-id>``
-     - Namespace ID
+   * - ``<namespace>``
+     - Namespace
 
-The ``<namespace-id>`` must be of the limited character set for namespaces, as 
+The ``<namespace>`` must be of the limited character set for namespaces, as 
```

Comment:
Should this be 

"The ``<namespace>`` must be the name of an existing namespace"

We can mention the limited character set in the "create section"